tools/bump_version.sh

```diff
@@ -22,20 +26,80 @@ update_image_versions() {
     sed -i "s/asterinas\/asterinas:[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+/asterinas\/asterinas:${new_version}/g" $1
 }
 
+# Print the help message
+print_help() {
+  echo "Usage: ./tools/bump_version.sh bump_type"
```

Comment:
Print an empty line here to separate the usage summary from detailed description.